content/en/docs/tutorials/security/cluster-level-pss.md

```diff
@@ -292,7 +292,7 @@ following:
    kubectl apply -f https://k8s.io/examples/security/example-baseline-pod.yaml
    ```
 
-   The output is similar to this:
+   The Pod does start OK; the output includes a warning. For example:
```

Comment:
Maybe instead:

"The pod is started normally, but the output includes a warning"
?